spell/spell-crafter-goerli-workflow.md

```diff
@@ -87,6 +87,20 @@ PR: https://github.com/makerdao/spells-goerli/pull/<TODO>
   * [ ] Use DssExecLib Functions
     * [ ] Ensure `DssExecLib` address used in current spell (`DssExecLib.address`) matches `dss-exec-lib` [Latest Release Tag](https://github.com/makerdao/dss-exec-lib/releases/latest)
     * [ ] Check previous spells for common patterns
+* [ ] Add comments to the spell:
```

Comment:
I would actually move it much higher, right after:
```
* [ ] Cleanup `src/test/config.sol`
```
(same for the reviewers checklist)

The reason are:
- Content sections are useful for the cleanup especially because they allow conflict-free contributions from third parties
- Review should always start with checking if instructions are correctly transferred to the code and then use them as the reference for the actual code. Because if instructions are incorrect (or outdated) then the code is most probably too